tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.md

```diff
@@ -0,0 +1,39 @@
+Test 6-4 - Verify vic-machine create basic use cases
+=======
+
+#Purpose:
+Verify vic-machine create basic usage
+
+#References:
+* vic-machine-linux create -h
+
+#Environment:
+This test requires that a vSphere server is running and available
+
+#Test Steps:
+1. Issue the following command:
+```
+vic-machine create --name=<VCH_NAME> --target=<TEST_URL> \
+    --user=<TEST_USERNAME> --image-datastore=<TEST_DATASTORE> --password=<TEST_PASSWORD> \
+    --bridge-network=<NETWORK> --compute-resource=<TEST_RESOURCE>
+```
+2. Run regression tests
+3. Remove the VCH
+4. Issue the following command:
+```
+vic-machine-linux create --name=<VCH_NAME> --target="<TEST_USERNAME>:<TEST_PASSWORD>@<TEST_URL>" \
+    --image-datastore=<TEST_DATASTORE>
+```
+5. Run regression tests
+6. Remove the VCH
+7. Issue the following command:
+```
+vic-machine-linux create --name=<VCH_NAME> --target=<TEST_URL> --user=<TEST_USERNAME> --image-datastore=<TEST_DATASTORE> --appliance-iso=bin/appliance.iso --bootstrap-iso=bin/bootstrap.iso --password=<TEST_PASSWORD> --force=true --bridge-network=network --compute-resource=<TEST_RESOURCE> --timeout <TEST_TIMEOUT> --volume-store=<TEST_DATASTORE>/test:default
+```
+8. Run regression tests
+9. Remove the VCH
+
```

Comment:
Can we get a couple of negative test cases here? At least a few that just attempt to supply and invalid resource? Like a bad compute-resource, or bad bridge-network and if there is any required param try not supplying it, and supplying a parameter that doesn't exist.